core/src/main/scala/kafka/controller/ControllerContext.scala

```diff
@@ -23,18 +23,11 @@ import org.apache.kafka.common.TopicPartition
 import scala.collection.{Map, Seq, Set, mutable}
 
 object ReplicaAssignment {
-  def fromOldAndNewReplicas(oldReplicas: Seq[Int], newReplicas: Seq[Int]): ReplicaAssignment = {
-    val fullReplicaSet = (newReplicas ++ oldReplicas).distinct
-    ReplicaAssignment(
-      fullReplicaSet,
-      fullReplicaSet.diff(oldReplicas),
-      fullReplicaSet.diff(newReplicas)
-    )
-  }
```

Comment:
This method was not used by the controller after we introduced `reassignTo`. This functionality was moved to `reassignTo`.